src/current-refinements/current-refinements.ts

```diff
@@ -1,17 +1,28 @@
 import { Component, Input, Inject, forwardRef } from '@angular/core';
 
-import { connectCurrentRefinedValues } from 'instantsearch.js/es/connectors';
+import { connectCurrentRefinements } from 'instantsearch.js/es/connectors';
 import { BaseWidget } from '../base-widget';
 import { NgAisInstantSearch } from '../instantsearch/instantsearch';
-import { noop, capitalize } from '../utils';
+import { noop } from '../utils';
+
+export type CurrentRefinementsItem = {
+  attribute: string;
+  refine: Function;
+  refinements: {
+    type: string;
+    attribute: string;
```

Comment:
We could avoid the optional properties. We could create multiple types for each of the available refinement, `numeric`, `disjunctive`, etc... Then we can use the `attribute` property [to detect](https://basarat.gitbooks.io/typescript/docs/types/discriminated-unions.html) which kind of refinement we have.

Reply:
I added a TODO for this one